pkg/util/platform_status.go

```diff
@@ -51,7 +54,7 @@ func GetPlatformStatus(client client.Client, infra *configv1.Infrastructure) (*c
 		return nil, fmt.Errorf("invalid install-config: %v\njson:\n%s", err, data)
 	}
 	return &configv1.PlatformStatus{
-		Type: infra.Status.Platform,
+		Type: status.Type,
```

Comment:
If we needed to "build a platform status from the deprecated install-config" as mentioned in L35, why not hard-code this `Type` to the AWSPlatformType?

Reply:
Yes, it makes sense. I've applied it.